common/migrations/0031_room_class_room.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 4.2.16 on 2025-11-22 17:00
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('common', '0030_llmreviewprompt_suggestedcomment'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Room',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('number', models.CharField(max_length=30)),
```

Comment:
It's weird that it is called "number", but it is a string. Maybe `code`?

Reply:
OK, changed.